packages/new-auth-ui/serverless.yml

```diff
@@ -0,0 +1,118 @@
+# https://www.npmjs.com/package/serverless-single-page-app-plugin
+service: cloud-developer-portal
```

Comment:
Needs updating or you are going to nuke the dev portal!

Reply:
ah balls, need to change to CDK